shared/packages/expectationManager/src/internalManager/lib/trackedWorkerAgents.ts

```diff
@@ -123,7 +123,7 @@ export class TrackedWorkerAgents {
 						countQueried++
 						const cost = await workerAgent.api.getCostForExpectation(trackedExp.exp)
 
-						if (cost.cost < Number.POSITIVE_INFINITY) {
+						if (cost.cost !== null) {
```

Comment:
I think a comment that this means "less than infinite" would be nice
```suggestion
						if (cost.cost !== null) {
						// null means that the cost is "infinite"
```